src/wasm-lib/Cargo.toml

```diff
@@ -79,7 +79,10 @@ kittycad = { version = "0.3.28", default-features = false, features = ["js", "re
 kittycad-modeling-cmds = { version = "0.2.77", features = ["websocket"] }
 
 [workspace.lints.clippy]
+assertions_on_result_states = "warn"
+dbg_macro = "warn"
```

Comment:
It would be nice if we could have this run only on ci, but I don't think there's a good way to do that

Reply:
Cargo clippy is a separate check/workflow from cargo test, so dbg!() should run in CI.